charmcraft/application/commands/store.py

```diff
@@ -1037,6 +1037,11 @@ def fill_parser(self, parser: "ArgumentParser") -> None:
 
     def run(self, parsed_args: "Namespace") -> None:
         """Run the command."""
+        emit.progress(
```

Comment:
question: A test exists for the deprecated `register-bundle` command. Do you want to add one for this?

Reply:
Nah, because it's going away in a couple of months anyway.